### DISCUSSION TITLE
why method isCloneMode() in *Dialog.ts Always return false

### DISCUSSION THREAD

**Discussion by choss21** [serenity-is/Serenity#5925] [Help]
Hello everybody 

I have a question

in file [Serenity\src\Serenity.Scripts\CoreLib\UI\Dialogs\EntityDialog.ts method isCloneMode()](https://github.com/serenity-is/Serenity/blob/523c0195fec17089e1dd7e812650c2edaf47a7ac/src/Serenity.Scripts/CoreLib/UI/Dialogs/EntityDialog.ts)  always return false, 

I am applying the clone functionality, but I cannot determine if the dialog is in Clone mode 

![image](https://user-images.githubusercontent.com/13734677/129494714-ff95e979-f7ec-4bee-b90e-1f24fb465767.png)

**Comment by VictorTomaili** [comment]
You need to override clone detection per your business or send an option to your sub dialog
